Tests/StorageProviderTests.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  ContainerTests.swift
+//  Holocron-iOS
+//
+//  Created by Will McGinty on 3/19/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Foundation
+@testable import Holocron
+import KeychainAccess
+import XCTest
+
+extension String: RawRepresentable {
+    public var rawValue: String { return self }
+    
+    public init?(rawValue: String) {
+        return nil
+    }
+}
+
+struct StorageProviderTester {
```

Comment:
Can you add some `// MARK`s to help distinguish this struct, the MockStorageProvider class, and the test cases.